hack/make-rules/build_with_container.sh

```diff
@@ -26,9 +26,17 @@ KUBEEDGE_BUILD_IMAGE=${KUBEEDGE_BUILD_IMAGE:-"kubeedge/build-tools"}
 DOCKER_GID="${DOCKER_GID:-$(grep '^docker:' /etc/group | cut -f3 -d:)}"
 
 echo "start building inside container"
-docker run --rm -u "${UID}:${DOCKER_GID}" \
+set +u
+# If in Github Action workflow or Travis CI, we disable -it option, or they will report error: the input device is not a TTY
+# In Github Action, environment variable GITHUB_ACTIONS is always set to true
+# In Travis CI, environment variable TRAVIS is always set to true
+if [[ $GITHUB_ACTIONS != true && $TRAVIS != true ]]; then
```

Comment:
```suggestion
if [[ $GITHUB_ACTIONS = true || $TRAVIS = true ]]; then
```
for ci scenario, disable it.

Reply:
fixed all above